src/components/ae-card/ae-card.md

```diff
@@ -10,7 +10,27 @@
     <ae-text contenteditable>My Account</ae-text>
   </template>
   <template slot="header">
-    <ae-text fill="white" face="mono-base">20.65 AE</ae-text>
+    <ae-dropdown>
+      <ae-icon name="more" size="20px" slot="button" />
+      <li>
+        <ae-button>
+          <ae-icon name="copy" />
+          Copy Address
+        </ae-button>
+      </li>
+      <li>
+        <ae-button>
+          <ae-icon name="share" />
+          Share Address
+        </ae-button>
+      </li>
+      <li>
+        <ae-button>
+          <ae-icon name="eye" />
+          Details
+        </ae-button>
+      </li>
+    </ae-dropdown>
```

Comment:
Leave just an AeIcon, examples of AeDropdown should be in the documentation for AeDropdown component.

Reply:
I've added different ways to display the dropdown content. 

- With Icon
- With button
- Without anything simply `li` element

Reply:
> I've added different ways to display the dropdown content.

please, do this in the documentation for **AeDropdown** component